recipe/meta.yaml

```diff
@@ -1,41 +1,47 @@
-{% set version = "1.1.7" %}
+{% set version = "2.0" %}
 
 package:
   name: pygmo
   version: {{ version }}
 
 source:
-  fn: pagmo-{{ version }}.tar.gz
-  url: https://github.com/esa/pagmo/archive/{{ version }}.tar.gz
-  sha256: 6d8fab89ef9d5d5f30f148225bf9b84b2e5a38997f3d68b85547840e9fd95172
+  url: https://github.com/esa/pagmo2/archive/v{{ version }}.tar.gz
+  sha256: 022af6bad2f1d083c2cacf4b121722541df80b6f168519a8eba0709cef02163a
 
 build:
-  number: 2
-
-  # https://github.com/esa/pagmo/issues/94
-  skip: true  # [win]
+  number: 0
+  skip: true  # [(win and py27) or win32]
 
 requirements:
   build:
     - toolchain
-    - python
     - cmake
-    - boost 1.63.*
+    - eigen
+    - nlopt
+    - boost     1.63.*
+    - numpy     x.x      # [not osx]
+    - numpy     1.12.*   # [osx]
```

Comment:
On the staged-recipes repo, it resulted in a timeout on travis as it would take too long to build all the variants. On the feedstock CI, are the OSX builds for the different numpy versions split in different containers or are they part of the same individual CI run? In the latter case, it seems like removing this workaround would double the CI runtime per single instance, up to circa 50 minutes. This would seem dangerously close to timeout.